concepts/framework/http_cache.md

```diff
@@ -47,20 +47,35 @@ Set the defaults value of the `_httpCache` key to `true`. Examples for this can
 public function index(SalesChannelContext $context, Request $request): Response
 ```
 
-### Cache Cookies
+### Determining the cache key
 
-Shopware uses several client cookies to differentiate the requests. This allows us to save the cache differentiated between clients, e.g., different customer groups or different active rules.
+Determining the cache key is one of the most important tasks of the HTTP cache. The cache key is used to identify a request and its corresponding response. If the same request comes in again, the cache key will be used to look up the corresponding response in the cache storage.
+For a dynamic system like Shopware, the cache key needs to take the application state into account, as the response to the same request might differ e.g., based on the tax state of the currently logged-in customer.
+At the same time, it needs to be possible to generate the cache key directly from the request to support reverse proxy caches, where the caching is handled by a standalone application that has no access to Shopware's internal application state.
+Shopware generates a `cache-hash` that encodes the application state and this hash is passed alongside every request and response, the caching component will then generate the exact cache key based on the cache-hash.
```

Comment:
Here it's referenced as `cache-hash` but below as `sw-cache-hash`. I guess here you are talking about the concept and want to differentiate, but I think maybe it makes sense to be consistent? Either that, or in the `sw-cache-hash` section specifically mention, eg "This is the cookie name used to store the `cache-hash` (mentioned above)"